app/controllers/passwords_controller.rb

```diff
@@ -5,7 +5,7 @@ def new
 
   def create
     PasswordReset.send_reset_instructions_to(params[:user][:email])
-    redirect_to new_session_path, notice: t('devise.passwords.send_instructions')
+    redirect_to new_session_path, notice: t('passwords.send_instructions')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.